Automattic/vip-go-mu-plugins
# Merge VIP Support Repo

## Description

Merge the [VIP Support Repo](https://github.com/Automattic/vip-support/) into this one. The maintenance overhead of having `vip-support` in a separate repo is prohibitively high for little gain, so let's combine them.

TODO:

- [x] Replace submodule with folder
- [x] Identify and fix any redundant or unused config / files
- [x] Move unit tests
- [x] Composer?

### VIP Support: Merge repository

This is a behind-the-scenes change to merge 2 repositories together for easier maintenance.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Add yourself as a support user
1. Go to WP Admin
1. Ensure you still have regular permissions and there are no errors
